tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,21 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
```

Comment:
I was just going by your comment here https://github.com/pouchdb/pouchdb/issues/5471#issuecomment-232240310 that said it should only run in Node. I can change it.